resources/views/account/order.blade.php

```diff
@@ -1 +1,50 @@
-Order
+@extends('rapidez-ct::account.partials.layout')
+
+@section('title', __('Order') . ' #' . $id)
+@section('button')
+    <graphql-mutation
+        query='mutation { reorderItems(orderNumber: "{{ request()->id }}") { cart { id } userInputErrors { message } } }'
+        redirect="/cart"
+        :callback="reorderCallback"
+    >
+        <form
+            slot-scope="{ mutate, mutating, mutated }"
+            v-on:submit.prevent="mutate"
+        >
+            <x-rapidez-ct::button.enhanced
+                class="flex items-center"
+                type="submit"
+                v-cloak
+            >
+                @lang('Order again')
+            </x-rapidez-ct::button.enhanced>
+        </form>
+    </graphql-mutation>
+@endsection
+@section('robots', 'NOINDEX,NOFOLLOW')
+
+@section('account-content')
+    <graphql
+        query='@include('rapidez::account.partials.queries.order')'
+        check="data.customer.orders.items[0]"
+        cache="orderdetail{{ $id }}"
+    >
+        <div
+            v-if="data"
+            slot-scope="{ data }"
```

Comment:
As explained previously, the graphql tag is a vue component and thus it must adhere to the single root element rule. If I replaced the div with a template, it would violate this rule because the template is not rendered in the DOM and would result in multiple elements.